src/main/java/org/apache/sling/maven/feature/launcher/StopMojo.java

```diff
@@ -37,9 +39,20 @@
     @Component
     private ProcessTracker processes;
     
+    /**
+     * If {@code true} stopping the server is deferred until you press the Enter key.
+     */
+    @Parameter(property = "feature-launcher.waitForInput", required = false, defaultValue = "false")
+    protected boolean waitForInput;
+
+    @Component
+    private Prompter prompter;
+
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
-        
+        if (waitForInput) {
```

Comment:
Can we make this conditional of being executed in an interactive mode? Then the error message for when the prompt fails could be cleared.

Reply:
Done in https://github.com/apache/sling-feature-launcher-maven-plugin/pull/1/commits/f5c6159cccbeecc3eedcd86fb8dffb723ba93ad8